walletd/src/key_pair.rs

```diff
@@ -34,11 +34,16 @@ pub enum MnemonicKeyPairType {
 /// This struct is used specify options for and build a [KeyPair] struct
 #[derive(Debug, Clone, PartialEq, Eq, Default)]
 pub struct KeyPairBuilder {
-    mnemonic_phrase: Option<String>,
-    mnemonic_seed: Option<Seed>,
-    passphrase: Option<String>,
-    network_type: HDNetworkType,
-    style: MnemonicKeyPairType,
+    /// Option to set the mnemomonic phrase
```

Comment:
These should not be made public to satisfy integration tests. they should be tested via unit tests where the values are accessible as private properties

Reply:
I can make them private again but then would need to add the methods for getting the values